src/test/java/com/emc/mongoose/storage/driver/pravega/integration/DataOperationsTest.java

```diff
@@ -135,15 +134,16 @@ public final void testCreateEvent() throws Exception {
 						0, OpType.CREATE, dataItem, null, streamName, credential, null, 0, null);
 
 		String scope = "goose";
+		val result = new ArrayList<DataOperation<DataItem>>(1);
+		val resultOut = new ListOutput<DataOperation<DataItem>>(result);
+		operationResultOutput(resultOut);
 		prepare(createTask);
 		createTask.status(Operation.Status.ACTIVE);
 		put(createTask);
-
-		DataOperation<DataItem> result = get();
-		while (result == null) {
-			result = get();
-		} // need to wait for operation to be executed
-		assertEquals(Operation.Status.SUCC, result.status());
+		while (result.isEmpty()) {
+			Thread.sleep(1);
```

Comment:
is this done so that we don't call isEmpty() too often?

Reply:
1 ms is very long time
the typical operation duration is usually tens of ms